test/functional/plugin/lsp_spec.lua

```diff
@@ -3519,6 +3519,27 @@ describe('LSP', function()
       local expected = { '```cs', 'TestEntity.TestEntity()', '```', 'some doc' }
       eq(expected, result)
     end)
+
+    it('higlights active parameters in multiline signature labels', function()
+      local _, hl = exec_lua(function()
+        local signature_help = {
+          activeSignature = 0,
+          signatures = {
+            {
+              activeParameter = 1,
+              label = 'fn bar(\n    _: void,\n    _: void,\n) void',
+              parameters = {
+                { label = '_: void' },
+                { label = '_: void' },
+              },
+            },
+          },
+        }
+        return vim.lsp.util.convert_signature_help_to_markdown_lines(signature_help, 'zig', { '(' })
+      end)
+      local expected = { 3, 4, 3, 11 }
```

Comment:
If I understood correctly, this refers to the second parameter (`activeParameter = 1` with zero-based index) => `_: void`; in line 3 (Row 2, with zero-based index). I was confused why it's not `{ 2, 4, 2, 11 }` -- Could you please clarify which index is exactly being used and document it if it would make it easier to understand?

It looks like we're using 1-based index here for line (ro~) or 0-based index for column (with end-exclusive), but `vim.highlight.range` seems to accept 0-based index for line/col (Range4 also implies 0-based row/col).

Reply:
Hmm, maybe I got it; looks like it's because of the additional line introduced as the code is wrapped with a codeblock: "```...```". So it *is* 0-index.

Reply:
@wookayin exactly! I actually left a small comment about that here:
https://github.com/neovim/neovim/blob/21911f2cc556e6b05ad58a92cc7d4539e6a939ae/runtime/lua/vim/lsp/util.lua#L900-L906

But I'll make sure to clarify this in the test too.

Reply:
And thank you for the review @wookayin <3!